.gitignore

```diff
@@ -42,3 +42,7 @@ npm-debug.log
 
 # Credential files
 /gcs-credentials.json
+
+# virtualenv
+venv/
```

Comment:
this should be in your global .gitignore not in this repo.

Reply:
How do you distinguish the two?